packages/loaders/openapi/tests/204-response-with-empty-content.test.ts

```
@@ -0,0 +1,18 @@
+import { GraphQLSchema } from 'graphql';
+import { printSchemaWithDirectives } from '@graphql-tools/utils';
+import { loadGraphQLSchemaFromOpenAPI } from '../src/loadGraphQLSchemaFromOpenAPI.js';
+
+describe('204 response with empty content', () => {
+  let createdSchema: GraphQLSchema;
+  beforeAll(async () => {
+    createdSchema = await loadGraphQLSchemaFromOpenAPI('test', {
+      source: './fixtures/204-response-with-empty-content.json',
+      cwd: __dirname,
+      ignoreErrorResponses: true,
```

Should we ignore error responses?